index.js

```diff
@@ -12,16 +12,16 @@ var Nightmare  = require('nightmare'),
 // bluebird's promises.
 Queue.configure(Promise);
 
-var nightmare = Nightmare(),
-    maxConcurrent = 1,
-    maxQueue = Infinity,
-    queue = new Queue(maxConcurrent, maxQueue);
+var concurrency = config.concurrency,
```

Comment:
could you add `process.env.CONCURRENCY || config.concurrency`? That way this is easier once the thing is in Docker (you do not need to edit config etc)